int32 current_state
int32 lane_number
float32 dist
